### ISSUE TITLE
Remove Gallery+ from appstore for 8.2?

### ISSUE THREAD

**Issue by ghost** [owncloud/gallery#482]
Hi,

as "Gallery+" is now shipped in 8.2 as "Gallery" as a core app is there any reason why Gallery+ is kept for 8.2 within the appstore at https://apps.owncloud.com/content/show.php/Gallery%2B?content=169116

The only difference i'm seeing here is that the appstore has Gallery+ 14.3.0 where Gallery in 8.2 is at version 14.2.0. Shouldn't be Gallery also shipped in version 14.3.0 in core?

**Comment by oparoz** [comment]
Gallery+ exist so that I can backport new features to versions of ownCloud which are in use right now.
The 8.2 version of Gallery+ contains fixes and features which will only be available in 9.0 and will always have a higher version number than the official Gallery.

Here are a few reasons for this setup:
- I never use the latest version of ownCloud in production
- I want the latest fixes and features (if possible) on what I run in production
- I only have time to cherry pick commits for 2 versions and I'm the only maintainer of both projects, so I picked Gallery+
- Gallery+ doesn't have the same requirements as the official version. I can drop support for old browsers, introduce advanced features, change the UI, etc. 

My main goal when merging my work back into core was to offer a stable, documented base on which to build, but I kept adding more features, so both versions are very similar currently and I'm one of the only person knowing the code inside out.

Nothing is set in stone and we'll see how this all evolves as more people get involved.
